src/shell/types.rs

```diff
@@ -124,11 +124,7 @@ impl ShellState {
       }
     } else {
       self.shell_vars.remove(&name);
-      if value.is_empty() {
-        self.env_vars.remove(&name);
```

Comment:
Maybe we should add an `unset` command to the shell for deleting environment variables?

Reply:
We'd need to probably add `UnsetVar(String)` here: https://github.com/denoland/deno_task_shell/blob/1f87c1f82025098c37bfce8c49dffb4a28ccc012/src/shell/types.rs#L152-L158

Reply:
Adding `unset` makes a lot of sense to me!

Reply:
Would you like to add it? I added this to dax here https://github.com/dsherret/dax/pull/134/files so it should be easy to copy and add it to this repo in rust.

Reply:
Sure, I'll work on that real quick.